compiler/tests-compiler/unix_fs.ml

```diff
@@ -19,8 +19,14 @@
 
 open Util
 
+(* Unix.LargeFile requires unix_stat_64 which causes tests
```

Comment:
I don't understand why this is only an issue on windows.
What about we just alias the primitive to unix_stat ?

Reply:
I was also confused by that. The type signatures look the same, so it would probably work. I should probably provide the `unix_lstat_64` too: https://github.com/ocaml/ocaml/blob/f68acd1a618ac54790a8347fad466084f15a9a9e/otherlibs/win32unix/unix.ml#L407-L408